sheetScript.gs

```diff
@@ -225,7 +225,7 @@ function getVideoIds(channelId, lastTimestamp) {
       }
     } catch (e) {
       Logger.log("Cannot search YouTube, ERROR: " + e.message);
-      break;
+      return [];
```

Comment:
This is consistent with the rest of the script. Runs, but returns an empty list.

Reply:
This was concurrent with @dudemario's changes from `return videoIds` to `break`. This change makes sure the log still writes to the sheet.

Reply:
Sounds reasonable, similar case for [line 287](https://github.com/Elijas/auto-youtube-subscription-playlist-2/blob/1b3cdef21780cdc6790bd19447b53e8eea465ca5/sheetScript.gs#L287)

Reply:
Changed it to `break`. Your solution is much better.